### PR TITLE
first fr translation pass

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: metadata/lang/fr.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+level: "Niveau"
+introductory: "Introduction"
+intermediate: "Intermediaire"
+advanced: "Avancé"
+
+supporting-materials: "supports"
+slides: "Diapositives"
+datasets: "Jeux de données"
+workflows: "Workflows"
+supporting-galaxies: "Disponible su ces instances de Galaxy"
```
Comment (abretaud) [inline metadata/lang/fr.yml:23]:
```suggestion
supporting-galaxies: "Disponible sur ces instances de Galaxy"
```

File: metadata/lang/fr.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+faqs: "Questions fréquentes"
+#Have questions about this tutorial? Check out the FAQ page for the Introduction to Galaxy Analyses topic to see if your question is listed there. If not, please ask your question on the GTN Gitter Channel or the Galaxy Help Forum: "¿Tienes preguntas sobre este tutorial? Consulta la página de preguntas frecuentes para ver si tu pregunta aparece allí. De lo contrario, haz tu pregunta en en el canal GTN de Gitter o en el Foro de ayuda de Galaxy"
+references: "Références"
+feedback: "Retours"
+feedback-text: "Utilisation de ce matériel en tant que formateur ?"
+feedback-link-text: "Partagez votre expérience"
+feedback-text-learner: "Avez-vous utilisé ce matériel en tant qu'apprenti ou étudiant ? Cliquez sur le formulaire ci-dessous pour nous laisser votre avis"
+
+
+glossary: "Glosaire"
```
Comment (abretaud) [inline metadata/lang/fr.yml:43]:
```suggestion
glossary: "Glossaire"
```

File: metadata/lang/fr.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+citing-tutorial: "Comment citer ce tutorial"
+congrats: "Felicitacions! Vous avez terminé avec succès ce tutoriel !"
```
Comment (abretaud) [inline metadata/lang/fr.yml:45]:
```suggestion
congrats: "Félicitations ! Vous avez terminé avec succès ce tutoriel !"
```

**Review by abretaud — COMMENTED** [review]
¡Muy bonito!

File: metadata/lang/fr.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# metadata box
+overview: "Descriptin générale"
```
Comment (abretaud) [inline metadata/lang/fr.yml:2]:
```suggestion
overview: "Description générale"
```

**Comment by yvanlebras** [comment]
THANK YOU Anthony

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
please merge if you think this is ready, then will rebase your other PR to include this, so we can test if everything works more easily. Thanks @yvanlebras!

**Pull request merged**